image-filters/image-filters.markdown

```diff
@@ -209,15 +205,15 @@ We don’t want our app to be a tiny grey window, so the two essential methods
 that we will start by overriding (implementing in our own class, instead of
```

Comment:
Cool; do you want me to rewrite this and push the changes to this PR? Or will you do it when you fix the image references? Either is fine by me. @MichaelDiBernardo